Frameworks/BlockchainCertificates/Issuer.swift

```diff
@@ -231,6 +212,7 @@ public struct Issuer {
     }
     
     
+    /// TODO (kim): is this used?
```

Comment:
Oh, but to answer your question, yes, it's used. It's so we can use `NSCoding` to save a list of Issuers to disk & load them again. Structs can't conform to `NSCoding` (only `class`es can), but providing a means of converting to-and-from a dictionary gets around that (since dictionaries are classes that support `NSCoding`)